test/integration/project/tsconfig.base.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "noImplicitAny" : true,
+        "noImplicitThis" : true,
+        "alwaysStrict" : true,
+        "strictNullChecks": true,
+        "luaTarget": "JIT"
```

Comment:
luatarget ist not a compilerOption and needs to be declared in the root of the config.json.